File: customview/src/main/java/dinson/customview/api/OneApi.java
package dinson.customview.api;


import dinson.customview.entity.one.DailyDetail;
import dinson.customview.entity.one.DailyList;
import io.reactivex.Flowable;
import io.reactivex.Observable;
import okhttp3.ResponseBody;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.Path;
import retrofit2.http.Query;

/**
 * one一个api
 */

public interface OneApi {

    @GET("http://v3.wufazhuce.com:8000/api/hp/idlist/0")
    Flowable<DailyList> getDaily();

    @GET("http://v3.wufazhuce.com:8000/api/hp/detail/{id}")
    Flowable<DailyDetail> getDetail(@Path("id") int id);
}

File: blingbling/src/main/java/dinson/customview/api/OneApi.kt
package dinson.customview.api


import dinson.customview.entity.one.DailyDetailResult
import dinson.customview.entity.one.DailyList
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * one一个api
 */

interface OneApi {

    @GET("http://v3.wufazhuce.com:8000/api/hp/idlist/0")
    fun loadDaily(): Observable<DailyList>

    @GET("http://v3.wufazhuce.com:8000/api/hp/detail/{id}")
    fun getDetail(@Path("id") id: Int): Observable<DailyDetailResult>
}
